### ISSUE TITLE
`noSecrets` reports string with spaces.

### ISSUE THREAD

**Issue by Conaclos** [biomejs/biome#3861]
### Environment information

```block
Main
```

### What happened?

The following code is reported by [noSecrets](https://biomejs.dev/linter/rules/no-secrets/).
```js
export const C = "\nconst DEFAULT_CONFIG = /* @__PURE__ */ bare.Config({})\n";
```

[Playground](https://biomejs.dev/playground/?lintRules=all&code=ZQB4AHAAbwByAHQAIABjAG8AbgBzAHQAIABDACAAPQAgACIAXABuAGMAbwBuAHMAdAAgAEQARQBGAEEAVQBMAFQAXwBDAE8ATgBGAEkARwAgAD0AIAAvACoAIABAAF8AXwBQAFUAUgBFAF8AXwAgACoALwAgAGIAYQByAGUALgBDAG8AbgBmAGkAZwAoAHsAfQApAFwAbgAiADsA)

### Expected result

The rule could exclude strings that contain spaces.

EDIT: Finally, I am not sure how to fix this. Because some secrets can be inside a spaced string.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by SaadBazaz** [comment]
Will have to study original no-secrets code better. Does it offer this within-string checking? 

 Because if remove the feature of checking secret within a larger string, we can easily fix this problem with spaces heuristic.
